src/c++/vdo/tests/OldRecoveryJournal_t1.c

```diff
@@ -141,6 +143,40 @@ static void generate(void)
   close(fd);
 }
 
+/**
+ * Allocate and read the VDO contents from disk into a character array.
+ **/
+static void readVDOFromDisk(const char *fileName)
+{
+  int fd;
+  VDO_ASSERT_SUCCESS(open_file(fileName, FU_READ_ONLY, &fd));
+
+  off_t vdoSize;
+  VDO_ASSERT_SUCCESS(get_open_file_size(fd, &vdoSize));
+
+  VDO_ASSERT_SUCCESS(vdo_allocate(vdoSize, char, __func__, &expectedData));
+  ssize_t bytesRead;
+  VDO_ASSERT_SUCCESS(logging_read(fd, expectedData, vdoSize, __func__,
+                                  &bytesRead));
+  CU_ASSERT_EQUAL(vdoSize, bytesRead);
+  close(fd);
+}
+
+/**
+ * Check if the mismatch detected for a given pbn is an acceptable mismatch.
+ *
+ * Implements MismatchChecker.
+ **/
+static void mismatchChecker(physical_block_number_t pbn,
+                            char *expectedBlock,
+                            char *actualBlock)
+{
+  /* There are no acceptable mismatches */
```

Comment:
This comment feels at odds with the  function comment at line 166. And you really don't need two comments. Update the function comment to describe this checker. (The 'Implements' note on line 168 is fine.)

Reply:
Fixed.